main/main.c

```diff
@@ -611,6 +611,7 @@ PHP_INI_BEGIN()
 #ifdef PHP_WIN32
 	STD_PHP_INI_BOOLEAN("windows.show_crt_warning",		"0",		PHP_INI_ALL,		OnUpdateBool,			windows_show_crt_warning,			php_core_globals,	core_globals)
 #endif
+	STD_PHP_INI_ENTRY("syslog.ascii_only",			"0",		 PHP_INI_SYSTEM,	OnUpdateBool,		syslog_ascii_only,		php_core_globals, 		core_globals)
```

Comment:
There was a discussion some days ago in this thread https://github.com/php/php-src/pull/2674#discussion_r132667281 where it turned out it the encoding is not really respected, specifically after this experiment is what was to see here https://github.com/php/php-src/pull/2674#discussion_r132779019. If you mean just about checking the encoding - default_charset might suffice for that already, so no reason to introduce a complication. For encoding conversion it's probably even more complicated to start here. In general, the patch as is seemed to be OK, both for fixing the initial newline splitting issue and securing some bits, the further handling should be a matter of a presumably more complicated implementation. Since it targets master, it were OK to have this patch at least for the start, then looking whether a complicated implementation could ever make it.

Thanks.